packages/core/src/baseclient.ts

```diff
@@ -297,8 +292,8 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
   /**
    * Sets up the integrations
    */
-  public setupIntegrations(): void {
-    if (this._isEnabled() && !this._integrationsInitialized) {
+  public setupIntegrations(forceInitialize?: boolean): void {
```

Comment:
l: Maybe it would be nicer to split this into to methods, than to have this argument:

```ts
public setupIntegrations(): void {
  if (this._isEnabled() && !this._integrationsInitialized) {
    this.initializeIntegrations(); // <-- naming TBD??
  }
}

public initializeIntegrations(): void {
  this._integrations = setupIntegrations(this, this._options.integrations);
  this._integrationsInitialized = true;
}
```

I guess the idea is that e.g. spotlight would initialize sentry integrations even if not enabled? So that could then just call `client.initializeIntegration()`?